spec/declaration.dd

```diff
@@ -398,6 +399,14 @@ t1.t v1;  // v1 is type int
 t2 v2;    // v2 is type int
 t3 v3;    // v3 is type int
 t4 v4;    // v4 is type int
+
+alias Fun = int(string p);
+int fun(string){return 0;}
+static assert(is(typeof(fun) == Fun));
+
+alias MemberFun1 = void() const;
+alias MemberFun2 = const void(); // leading storage classes are not part of the type
+static assert(is(MemberFun1 == MemberFun2));
```

Comment:
If the leading storage class is not part of the type, how can this be true?

Reply:
It's just like when you declare a function. You can but it's recommended to put them before the body ;).

Reply:
Perhaps it would be a bit more clear if we say: "leading storage classes are interpreted as member function attributes", unless I'm missing something.